vue/src/components/SnapshotMap.vue

```diff
@@ -9,137 +9,56 @@
-                    </template>
-                    <span>Click to delete all markers</span>
-                  </v-tooltip>
-                </div>
-              </div>
-          </div>
-          <div id="PostitNaming" v-if="addMarkerMode
-                  && markerSelection.includes('tooltip')">
-            <div class="row">
-              <div class="column" style="width: 150px;">
-                <h6> Enter node & <br>click on the map.</h6>
-              </div>
-              <div v-if="markerSelection == 'tooltip-text-green'"
-              class="column" style="width: 50px;"
-              v-on:click="addMarkerMode = !addMarkerMode;
-              markerSelection = ''">
-                <v-icon large color="green" > mdi-tooltip-text </v-icon>
-              </div>
-              <div v-if="markerSelection == 'tooltip-text-blue'"
-              class="column" style="width: 50px;"
-              v-on:click="addMarkerMode = !addMarkerMode;
-              markerSelection = ''">
-                <v-icon large color="blue" > mdi-tooltip-text </v-icon>
-              </div>
-              <div v-if="markerSelection == 'tooltip-text-yellow'"
-              class="column" style="width: 50px;"
-              v-on:click="addMarkerMode = !addMarkerMode;
-              markerSelection = ''">
-                <v-icon large color="yellow" > mdi-tooltip-text </v-icon>
-              </div>
-              <div class="column" style="width: 150px; font-size: 1.4em">
-                <input id="InputMarkerName"  v-model="newPostItNode"
-                    placeholder="Enter your Post-it note here"
-                >
-              </div>
-            </div>
-          </div>
+      <v-main class="pa-0" ref="mapbox">
+      <div
+        id="MarkerButtons"
+        style="position: absolute; top: 6em; right: 1.8em; z-index: 500"
+      >
+        <button
+          style="background-color: #543076; border-radius: 50px"
+          v-on:click="markerTools=!markerTools; showAnnotations(); map.closePopup();"
+        >
+          <v-icon large color="white"> mdi-brush </v-icon>
+        </button>
+        <v-card v-if="markerTools" style="position:absolute: ;width: 36px">
+          <button v-on:click="toggelMarkerSelection('marker')">
+            <v-icon large color="black"> mdi-map-marker </v-icon>
+          </button>
+          <button v-on:click="toggelMarkerSelection('polygon')">
+            <v-icon large color="black"> mdi-vector-polygon </v-icon>
+          </button>
+          <button v-on:click="toggelMarkerSelection('note')">
+            <v-icon large color="black"> mdi-note-outline </v-icon>
+          </button>
+          <button v-on:click="toggelMarkerSelection(''); hideAnnotations();
+          markerTools=!markerTools;">
+            <v-icon large color="black"> mdi-eye-off </v-icon>
+          </button>
+        </v-card>
+        <div v-if="markerTools" >
+          <v-card v-if="markerSelection == 'marker'"
```

Comment:
Would suggest to use [<v-tooltip>](https://vuetifyjs.com/en/components/tooltips/) here to avoid needing custom css and more visual consistency. Can be tied to the existing model props `marker`, `polygon` etc. via [`v-model`](https://vuetifyjs.com/en/components/tooltips/#visibility).

Reply:
With  https://github.com/cividi/spatial-data-package-platform/commit/df1784790abe89b5f5f4d8f5b129e64b2e717d14 I changed to v-tooltip